Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -346,98 +314,40 @@ def create_values_for_returned_dict(iocs: list, request_args: RequestArguments)
     if len(ipv6_formatted_indicators) > 0:
         ipv6_formatted_indicators = ips_to_ranges(ipv6_formatted_indicators, request_args.collapse_ips)
         formatted_indicators.extend(ipv6_formatted_indicators)
-    out_dict = {
-        EDL_VALUES_KEY: list_to_str(formatted_indicators, '\n'),
-        "current_iocs": iocs,
-        "last_limit": request_args.limit,
-        "last_offset": request_args.offset,
-        "drop_invalids": request_args.drop_invalids,
-        "url_port_stripping": request_args.url_port_stripping,
-        "collapse_ips": request_args.collapse_ips,
-        "last_query": request_args.query
-    }
-    return out_dict, len(formatted_indicators)
+    if len(formatted_indicators) == 0 and request_args.invalidate_empty_edl:
+        formatted_indicators.append('#')
```

Comment:
Note that in `route_edl_values` we calculate the size of the edl by counting lines and setting the `X-EDL-Size` header. This change will cause us to set the header to 1 even though the size is 0. Might want to move this logic into `route_edl_values` after counting the size.